rest_api/apps/records/views.py

```diff
@@ -105,17 +106,29 @@ def TestRecordCreate(request, format=None):
 	Receive data from client
 	"""
 	data = request.data
+	print(22222222)
 
 	json_data = json.dumps(data[0], ensure_ascii=False)
 	json_data = json.loads(json_data)
 	# obj = data[0].pgbench
 	# jsLoads = json.loads(data[0])
+	# print(json_data)
 
 	from django.db import transaction
 
 	try:
 		secret = request.META.get("HTTP_AUTHORIZATION")
-		ret = Machine.objects.filter(machine_secret=secret, state='A').get()
+		print(33333333)
+		# ret = Machine.objects.filter(machine_secret=secret, state='A').get()
+		try:
+			ret = Machine.objects.filter(machine_secret=secret, state='A').get()
+		except Exception as e:
```

Comment:
Can you try to look at all the current users, and get one with the right permission (hard-code it like gsoccamp could temprorily work, but is not a full solution)